docs/governance/community-guidelines/templates/features.yaml

```diff
@@ -0,0 +1,10 @@
+title: CCC <Service Category> Security Threats
```

Comment:
```suggestion
title: CCC <Service Category> Features
```

Reply:
Should I commit this? And did you see the similar suggestion on the controls file?

Reply:
Sorry, multitasking all over the place! I threw that on so I didn't lose track of it.